js/docx/smoke/api_uni_color/to_json.js

```diff
@@ -0,0 +1,29 @@
+builder.CreateFile("docx");
+let oDocument = Api.GetDocument();
+let oParagraph = oDocument.GetElement(0);
+let oRGBColor = Api.CreateRGBColor(255, 111, 61);
+let jsonRGBColor = oRGBColor.ToJSON();
+oParagraph.AddText(jsonRGBColor);
+GlobalVariable["JSON_RGBColor"] = jsonRGBColor;
+builder.CloseFile();
+
+/////////////////////
+
+builder.CreateFile("docx");
+let oDocument = Api.GetDocument();
+let oParagraph1 = oDocument.GetElement(0);
+let jsonRGBColor = GlobalVariable["JSON_RGBColor"];
+let oRGBColor = Api.FromJSON(jsonRGBColor);
+oParagraph1.AddText(jsonRGBColor);
+oDocument.Push(oParagraph1);
+let oParagraph2 = Api.CreateParagraph();
+let oGs1 = Api.CreateGradientStop(oRGBColor, 0);
```

Comment:
Perhaps to simplify the script not to use gradient descent for color, but to set SolidFill?

Reply:
fix in new commit